frontend/encore/simple-example.rst

```diff
@@ -74,6 +74,21 @@ To build the assets, run:
 
     Stop and restart ``encore`` each time you update your ``webpack.config.js`` file.
 
+.. note::
+
+    If you prefer npm, run these commands instead:
```

Comment:
Maybe add a choice sentence above, like for ex:
 « Symfony is not linked to either Yarn neither NPM package manager for your frontend assets, choose the one you are fluent with, and use those command line respectively »
With link to both Yarn and NPM website
That way, it can avoid duplicating the code block and devs can understand they can use both command line for same output :)